### DISCUSSION TITLE
We want YOU to create new Clockwork apps!

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#35] [Announcements]
We're inviting developers with at least some experience to create Clockwork apps!

In anticipation of the launch of Clockwork v2, we want to make sure we have apps for people to use. So, we're inviting developers to beta-test Clockwork and create new apps!

Email us <a href="mailto:toasterpanic@gmail.com?subject=Request%20to%20Clockwork%20V2%20Beta%20Test&body=%2F%2F%2F%2F%20DO%20NOT%20EDIT%20THESE%20TITLES%20(signified%20with%20%2F%2F%2F%2F%20at%20the%20start%20of%20a%20lime)%0D%0A%2F%2F%2F%2F%20Put%20your%20answers%20under%20the%20titles%20of%20the%20questions.%0D%0A%0D%0A%2F%2F%2F%2F%20Why%20would%20you%20like%20to%20join%20the%20beta%20test%3F%0D%0A%0D%0A%2F%2F%2F%2F%20Do%20you%20have%20any%20prior%20experience%20with%20Clockwork%3F%0D%0A%0D%0A%2F%2F%2F%2F%20Do%20you%20have%20any%20prior%20experience%20with%20programming%3F%0D%0A%0D%0A%2F%2F%2F%2F%20Do%20you%20have%20a%20Github%2FGitlab%20account%3F%20Put%20the%20username%20here%20if%20so.%0D%0A%0D%0A%2F%2F%2F%2F%20Do%20you%20have%20a%20Discord%20account%3F%20If%20so%2C%20put%20your%20username%20%2B%20tag%20here%20(do%20not%20make%20a%20discord%20account%20if%20you're%20under%2013!)%0D%0A%0D%0A%2F%2F%2F%2F%20Do%20you%20have%20a%20Muenster%20account%3F%20If%20so%2C%20put%20your%20username%20here.%0D%0A">here</a> if you're interested. Thank you!

**Comment by adonzone** [comment]
sure

**Reply by simpansoftware** [reply]
why not

**Reply by adonzone** [reply]
yeah
